apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```

Comment:
there needs to be instructions somewhere for how to set this up

Reply:
https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md 
I have initially followed this instructions

Reply:
that should be linked to in the chart documentation, with probably an example command

Reply:
https://github.com/hmcts/chart-postgresql/pull/4